### PR TITLE
Paginate GCN events

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds pagination to the GCN events.

### CODE REVIEW

File: skyportal/handlers/api/gcn.py [hunk 1]
```diff
@@ -184,6 +184,10 @@ def get(self, dateobs=None):
               application/json:
                 schema: Error
         """
+
+        page_number = int(self.get_query_argument("pageNumber", 1))
+        n_per_page = int(self.get_query_argument("numPerPage", 100))
```
Comment (guynir42) [inline skyportal/handlers/api/gcn.py:189]: Add try catch with self error in case inputs are not integers.

Comment (guynir42) [inline skyportal/handlers/api/gcn.py:189]: Add try catch with self error in case inputs are not integers.

**Review by guynir42 — APPROVED** [review]
LGTM

File: skyportal/handlers/api/gcn.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py]: done

Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py]: done

**Pull request merged**